styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -336,29 +363,29 @@ private boolean isFlyteExecRelatedToRunState(AnnotatedFlyteExecutionId annotated
     return true;
   }
 
-  @VisibleForTesting
-  static Map<String, String> getExtraDefaultInputs(WorkflowInstance workflowInstance, StateData stateData) {
+  private static Map<String, String> getStyxVariables(WorkflowInstance workflowInstance, StateData stateData) {
     var triggerType = stateData.trigger().map(TriggerUtil::triggerType);
     var triggerId = stateData.trigger().map(TriggerUtil::triggerId);
 
     var mapBuilder = ImmutableMap.<String, String>builder();
 
-    mapBuilder.put("STYX_COMPONENT_ID", workflowInstance.workflowId().componentId());
-    mapBuilder.put("STYX_WORKFLOW_ID", workflowInstance.workflowId().id());
-    mapBuilder.put("STYX_PARAMETER", workflowInstance.parameter());
-    stateData.executionId().ifPresent(value -> mapBuilder.put("STYX_EXECUTION_ID", value));
-    triggerId.ifPresent(value -> mapBuilder.put("STYX_TRIGGER_ID", value));
-    triggerType.ifPresent(value -> mapBuilder.put("STYX_TRIGGER_TYPE", value));
-    // add trigger parameters but avoid any STYX_XXX ones to prevent collisions
-    stateData.triggerParameters().ifPresent(triggerParameters ->
-        triggerParameters.env().entrySet().stream()
-            .filter(entry -> !entry.getKey().startsWith("STYX_"))
-            .forEach(entry -> mapBuilder.put(entry.getKey(), entry.getValue()))
-    );
+    mapBuilder.put(STYX_COMPONENT_ID, workflowInstance.workflowId().componentId());
+    mapBuilder.put(STYX_WORKFLOW_ID, workflowInstance.workflowId().id());
+    mapBuilder.put(STYX_PARAMETER, workflowInstance.parameter());
+    stateData.executionId().ifPresent(value -> mapBuilder.put(STYX_EXECUTION_ID, value));
+    triggerId.ifPresent(value -> mapBuilder.put(STYX_TRIGGER_ID, value));
+    triggerType.ifPresent(value -> mapBuilder.put(STYX_TRIGGER_TYPE, value));
 
     return mapBuilder.build();
   }
 
+  // returns trigger parameters but avoid any STYX_XXX ones to prevent collisions
+  private static Map<String, String> getFilteredTriggerParams(TriggerParameters triggerParameters) {
+    return triggerParameters.env().entrySet().stream()
```

Comment:
Something not related to this PR but it looks we are using the environment variables i the `triggerParameters`. This is fine for now but it looks a bit confusion. Maybe this is something we need to re-evaluate later on.